### PR TITLE
fix(iam): move to full ARN for stability

### PR DESCRIPTION
**Pull request by MickVanDuijn**

### CODE REVIEW

**Comment by MickVanDuijn** [comment]
Current dependencies on/for this PR:
* main
  * **PR #12** <a href="https://app.graphite.dev/github/pr/skyleague/aws-rest-api/12" target="_blank">Graphite</a>  👈
    * **PR #13** <a href="https://app.graphite.dev/github/pr/skyleague/aws-rest-api/13" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/aws-rest-api/12?utm_source=stack-comment).

**Review by pevisscher — APPROVED** [review]

**Comment by MickVanDuijn** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/aws-rest-api/12?utm_source=stack-comment).


Job Reference: a1jrMWh6tXCahktFLQju

**Pull request merged**
